### PR TITLE
Adds property model

### PR DESCRIPTION
**Pull request by lellky**
I got a list of models from Flexit. It is really simple to extract the model from the serial number (the first six digits represents the model, the last six digits is just a consecutive number/serial number)

This is just a proposal.

The list with models are translated in Swedish, Norwegian, English and Finnish. If we want to translate the models that code should probably live in Home Assistant core.

Example output:
```
Device Name: HvacFnct21y_A
Serial Number: 800130-002967
Device Model: Air handling unit Nordic S4 RER
Outside air temp.: -1.4 °C
Supply air temp.: 17.9 °C
Extract air temp.: 20.0 °C
Exhaust air temp.: 2.9 °C
Room air temp.: 15.1 °C
Room 1 humidity: 0.0%
Room 2 humidity: 0.0%
Room 3 humidity: 0.0%
Comfort button state: True
Operation mode: 3
Ventilation mode: 3
Air temp. setpoint Away: 18.0 °C
Air temp. setpoint Home: 19.0 °C
Fireplace duration remaining: 10 minutes
Rapid ventilation duration remaining: 10 minutes
Supply air fan control signal: 74%
Supply air fan RPM: 2824
Exhaust air fan control signal: 70%
Exhaust air fan RPM: 2652
Electric heater enabled: True
Electric heater nominal power: 0.800000011920929 kW
Electric heater power consumption: 0.14538198709487915 kW
Fan setpoint - supply air Home: 74%
Fan setpoint - supply air Away: 42%
Fan setpoint - supply air High: 100%
Fan setpoint - supply air Cooker: 50%
Fan setpoint - supply air Fireplace: 90%
Fan setpoint - extract air Home: 70%
Fan setpoint - extract air Away: 40%
Fan setpoint - extract air High: 100%
Fan setpoint - extract air Cooker: 50%
Fan setpoint - extract air Fireplace: 50%
Air filter operating time: 8712.0 hours
Air filter exchange interval: 8784.0 days
Air filter polluted: False
Heat-exchanger efficiency: 80%
Heat-exchanger speed: 100%
```

### CODE REVIEW

File: flexit_bacnet/nordic.py [hunk 1]
```diff
@@ -163,3 +163,22 @@
 DEVICE_PROPERTIES = [
     item for _, item in globals().items() if isinstance(item, DeviceProperty)
 ]
+
+# The first six digits in the Nordic serial number corresponds to the Nordic model.
+NORDIC_MODELS = {
+    800111: "Air handling unit Nordic S2 REL",
```
Comment (piotrbulinski) [inline flexit_bacnet/nordic.py:169]: I would keep it short: "Nordic S2 REL"

Comment (lellky) [reply flexit_bacnet/nordic.py:169]: That is a good suggestion!

**Comment by lellky** [comment]
Actually, I think I was too fast. We should probably remove "Nordic" from the model name as well. Then the device could be named "Flexit Nordic" in Home Assistant and the model would be only "S4 REL" for example.

**Comment by lellky** [comment]
I might have been to fast again... I think it is very important we get the naming right.

The integration in Home Assistant is called Flexit (flexit_bacnet)
The name of the device should then be either "Flexit Nordic" or just "Nordic"
The model of the device should then be only "S4 REL"

What I want is for a climate entity to be called (in HA): `climate.flexit_nordic_s4_rel` or `climate.nordic_s4_rel` and then a temperature sensor would be `sensor.flexit_nordic_s4_rel_room_tempterature` or `sensor.nordic_s4_rel_room_temperature`

What do you think?

File: flexit_bacnet/nordic.py [hunk 2]
```diff
@@ -163,3 +163,26 @@
...
+    800211: "CL2 REL",
+    800200: "CL3 RER",
+    800201: "CL3 REL",
+    800300: "KS3 RER",
+    800301: "KS3 REL",
+}
+
+# The name of the device
+DEVICE_NAMES = {
+    "HvacFnct21y_A": "Flexit Nordic",
```
Comment (lellky) [inline flexit_bacnet/nordic.py:187]: Could probably be just "Nordic". We have to figure out if the device name can include the platform name (in Home Assistant).

**Pull request merged**